packages/shared/src/components/utilities/DateFormat.spec.tsx

```diff
@@ -0,0 +1,135 @@
+    });
+    const element = screen.getByRole('time');
+    expect(element).toHaveTextContent('Aug 01, 2021');
+  });
+
+  it('should render post time format as now', async () => {
+    const date = new Date();
+    renderComponent({ date, type: TimeFormatType.Post });
+    const element = screen.getByRole('time');
+    expect(element).toHaveTextContent('Now');
+  });
+
+  it('should render post time format as today', async () => {
+    const date = new Date();
+    renderComponent({
+      date: date.setHours(date.getHours() - 2),
+      type: TimeFormatType.Post,
+    });
+    const element = screen.getByRole('time');
+    expect(element).toHaveTextContent('Today');
+  });
+
+  it('should render post time format as yesterday', async () => {
+    const date = new Date();
+    renderComponent({
+      date: date.setDate(date.getDate() - 1),
+      type: TimeFormatType.Post,
+    });
+    const element = screen.getByRole('time');
+    expect(element).toHaveTextContent('Yesterday');
+  });
+});
+
+describe('date format comment type', () => {
+  it('should render comment time format', async () => {
+    renderComponent({
+      date: '2021-08-01T00:00:00Z',
+      type: TimeFormatType.Comment,
+    });
+    const element = screen.getByRole('time');
+    expect(element).toHaveTextContent('Aug 1, 2021');
+  });
+
+  it('should render comment time format as now', async () => {
+    const date = new Date();
+    renderComponent({
+      date,
+      type: TimeFormatType.Comment,
+    });
+    const element = screen.getByRole('time');
+    expect(element).toHaveTextContent('Now');
+  });
+
+  it('should render comment time format as minutes', async () => {
+    const date = new Date();
+    renderComponent({
+      date: date.setMinutes(date.getMinutes() - 2),
+      type: TimeFormatType.Comment,
+    });
+    const element = screen.getByRole('time');
+    expect(element).toHaveTextContent('2 mins');
+  });
+
+  it('should render comment time format as hours', async () => {
+    const date = new Date();
+    renderComponent({
+      date: date.setHours(date.getHours() - 2),
+      type: TimeFormatType.Comment,
+    });
+    const element = screen.getByRole('time');
+    expect(element).toHaveTextContent('2 hrs');
+  });
+});
+
+describe('date format read history type', () => {
+  it('should render comment time format', async () => {
+    renderComponent({
+      date: '2021-08-01T00:00:00Z',
+      type: TimeFormatType.ReadHistory,
+    });
+    const element = screen.getByRole('time');
+    expect(element).toHaveTextContent('Sun, 1 Aug 2021');
+  });
+
+  it('should render comment time format as today', async () => {
+    const date = new Date();
+    renderComponent({
+      date: date.setMinutes(date.getMinutes() - 2),
+      type: TimeFormatType.ReadHistory,
+    });
+    const element = screen.getByRole('time');
+    expect(element).toHaveTextContent('Today');
+  });
+
+  it('should render comment time format as hours', async () => {
```

Comment:
I see we check `expect(element).toHaveTextContent('Yesterday');` which doesn't align with the title.

Reply:
Copy paste issue, fixed :D